examples/sapper-template/install.js

```diff
@@ -0,0 +1,56 @@
+'use strict'
+
+const replace = require('replace')
+const degit = require('degit')
+const { writeFileSync, writeFile } = require('fs')
+const { join } = require('path')
+
+async function clone () {
+  const emitter = degit('sveltejs/sapper-template', {
+    force: true
+  })
+
+  emitter.on('info', info => {
+    console.log(info.message)
+  })
+
+  return emitter.clone('.')
+}
+
+async function patch () {
+  replace({
+    regex: /polka\(\)/,
+    replacement: 'export default polka()',
```

Comment:
Not sure if polka changed but this won't work. The template did polka.listen which does not return the server instance, it returns a promise instead.
I would suggest `';(module.exports=polka())` instead

Reply:
It definitely works, I've tried it a number of times. I'll verify exactly what is returned just to be sure, but If you try the template yourself, it works without issue as far as I can see.

Reply:
Tried again just to be sure:

```
$ vercel --prod
> Changelog: https://github.com/zeit/now/releases/tag/vercel@19.1.2
Vercel CLI 19.0.1
🔍  Inspect: https://vercel.com/dsrbl/sapper-template-vercel/3kjv4d34f [3s]
✅  Production: https://sapper-template-vercel.vercel.app [copied to clipboard] [30s]
```

https://sapper-template-vercel.vercel.app/

Reply:
Looks like this was 2 years ago https://github.com/lukeed/polka/issues/19 and 1 year ago merged in sapper-template not on purpose https://github.com/sveltejs/sapper-template/commit/355964717238cac799cecebec02fdc5c254b7775 😆